.github/workflows/benchmark.yml

```diff
@@ -47,6 +97,11 @@ jobs:
           auto-push: ${{ github.ref == 'refs/heads/main' }}
           comment-on-alert: true
           fail-on-alert: true
-          alert-threshold: '800%'
+          # Alerts when a 130% performance degradation occurs.
+          #
+          # There is 3.6% noise (maximum score is 3.68% greater than minimum score) for 30 runs.
+          # This noise is a little loud but negligible in most situations.
+          # ref: https://github.com/mizdra/eslint-interactive/commit/b751cfdef788ac6eb6b39d2d015494123cae51c1#comments
```

Comment:
This is a much larger value than the noise presented at https://pythonspeed.com/articles/consistent-benchmarking-in-ci/.

The detailed cause is unknown, but it is probably related to the fact that the process performed by eslint-interactive is dominated by I/O to the file system. valgrind does not virtualize I/O to the file system, so it is strongly affected by the CI noise.

Reply:
Nevertheless, previous benchmarks showed 300% noise. Given that, that's a huge improvement!

https://mizdra.github.io/eslint-interactive/dev/bench/
<img width="1060" alt="image" src="https://user-images.githubusercontent.com/9639995/218252097-afa62506-6825-482d-ab4e-9dafc383fc42.png">

Reply:
I rewrote the code to cause performance degradation as a test.

- diff
  - https://github.com/mizdra/eslint-interactive/compare/8c09f8e04c8ccd723c0d4e650ff5d1f9245a3157...65ac695388d3f2ae0e1fe25a9e6d4dd014858028
- before score: `21065268522` instructions
  - https://github.com/mizdra/eslint-interactive/commit/8c09f8e04c8ccd723c0d4e650ff5d1f9245a3157#comments
- after score: `33588677349` instructions
  - https://github.com/mizdra/eslint-interactive/commit/65ac695388d3f2ae0e1fe25a9e6d4dd014858028#comments

The score increased by 159.4 %. This satisfies the conditions for an alert to be triggered.